pkg/kubelet/images/image_gc_manager.go

```diff
@@ -149,6 +166,8 @@ type imageRecord struct {
 
 	// Pinned status of the image
 	pinned bool
+
+	exempted bool
```

Comment:
I would prefer if we can re-use `pinned` here. Having two fields for very similar purposes seems confusing to users.

Reply:
It is kept intentionally different as the current meaning associated with `pinned` is entirely different and this is driven as a CRI attribute driven state during the GC workflow. Exempted clearly annotates that this has been excluded from GC'ing by an external configuration list provided to the Kubelet.  Merging exempted and pinned would violate the meaning associated with the attribute. If we are to merge, we could rather rename both the fields and name call it as `ignoreGC` may be ?